src/commands/cmd_mount.rs

```diff
@@ -124,6 +124,24 @@ fn get_devices_by_uuid(uuid: Uuid) -> anyhow::Result<Vec<(PathBuf, bch_sb_handle
     Ok(devs)
 }
 
+fn get_uuid_for_dev_node(device: &std::path::PathBuf) ->  anyhow::Result<Option<Uuid>> {
+    let mut udev = udev::Enumerator::new()?;
+    udev.match_subsystem("block")?;
+
+    for dev in udev.scan_devices()?.into_iter() {
+        if let Some(devnode) = dev.devnode() {
+            if devnode == device {
+                let devnode_owned = devnode.to_owned();
+                let sb_result = read_super_silent(&devnode_owned);
+                if let Ok(sb) = sb_result {
+                    return Ok(Some(sb.sb().uuid()));
+                }
+            }
+        }
+    }
+    Ok(None)
```

Comment:
I took the approach that the code should display the same error message when it can't find something that the user supply.  That message before this change is `Fatal error: No device found from specified parameters`.  By returning `Ok(None)` we preserve the same error path progression which reports this error.  We could certainly change this to return instead.